src/style.css

```diff
@@ -1,3 +1,73 @@
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap");
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.nav {
+  box-shadow: 0 2px 4px 0 rgba(0,0,0, 0.3);
+}
+
+.nav__wrapper {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav__logo {
+  align-self: center;
+  justify-content: center;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav__link {
+  display: flex;
+  text-align: center;
+  align-items: center;
+  line-height: 14px;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-weight: 500;
+  font-size: 12px;
+  height: 60px;
+  color: #000;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.nav__link:active {
+  color: #00acdc;
+  box-shadow: 0 4px 0 #00acdc;
+}
```

Comment:
It's not required in the task, so you can just remove it

but if you want to keep it, it should look the same as the line in `is-active` class:
![image](https://github.com/mate-academy/layout_moyo-header/assets/86923089/2e1bd1ac-d93d-4017-b5c1-1e8aaad235c0)